File: Day 63/classwork/cw2.py
# 2) https://www.codewars.com/kata/53dc23c68a0c93699800041d/train/python


def smash(words):
    res = ""
    for i in range(len(words)):
        if i != len(words) -1:
            res = res + words[i] + " "
        else:
                res = res + words[i]
    return res    

File: Day 157/classwork/cw2.js
// 2) https://www.codewars.com/kata/54bf85e3d5b56c7a05000cf9/train/javascript


var number=function(array){
    let res = []
    for(let i = 0; i < array.length; i++){
        res.push(`${[i + 1]}: ${array[i]}`)
    }
    return res
}


number([1, 2, 3, 4, 5,])